.github/workflows/wheels.yml

```diff
@@ -0,0 +1,109 @@
+name: Build and upload to PyPI
+
+# Build on every branch push, tag push, and pull request change:
+# on: [push, pull_request]
+on: [pull_request]
+# Alternatively, to publish when a (published) GitHub Release is created, use the following:
+# on:
+#   push:
+#   pull_request:
+#   release:
+#     types:
+#       - published
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # os: [ubuntu-20.04, windows-2019, macos-11]
+        os: [ubuntu-20.04]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Fake metadata  # Probably a bad idea
+        run: touch pyproject.toml
```

Comment:
Terrible hack to get things going, otherwise the compilation doesn't even start.